backend/src/akvo/lumen/lib/import/common.clj

```diff
@@ -49,3 +49,12 @@
                 {:multipleType (:multipleType q)
                  :multipleId (:multipleId q)})))))
        questions))
+
+(defn extract-first-and-merge
+  "extract every first group-responses into one `main` response"
+  [ns-responses]
+  (->> ns-responses
+       (group-by #(:ns (meta %)))
```

Comment:
In the other PR, we changed the DB column to "namespace". Do we want to do the same in the code?

Reply:
yep, i thought the same when i reviewed the code, not sure then but I'll change it in the next PR